src/native/managed/cdacreader/src/Contracts/Thread.cs

```diff
@@ -0,0 +1,58 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+
+namespace Microsoft.Diagnostics.DataContractReader.Contracts;
+
+// TODO: [cdac] Add other counts / threads
+public record struct ThreadStoreData(
```

Comment:
Does this need to be `public`?

Reply:
Is a `record` appropriate in this case? I assume we want this to be immutable and will have zero or few methods, but want to confirm.